### PR TITLE
add some controller specs

### PR DESCRIPTION
**Pull request by ericcf**

### CODE REVIEW

File: test/spec/controllers/main.js [hunk 1]
```diff
@@ -1,22 +0,0 @@
...
-
-  // Initialize the controller and a mock scope
-  beforeEach(inject(function ($controller, $rootScope) {
-    scope = $rootScope.$new();
-    MainCtrl = $controller('MainCtrl', {
-      $scope: scope
-    });
-  }));
-  it('should have the app name', function() {
-    expect(scope.appTitle).toBe('FOCUS');
```
Comment (michaelwehrley) [inline test/spec/controllers/main.js]: No dice? Test didn't make the cut? :scissors:

File: test/unit/controllers/MainControllerSpec.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      fetchEncryptedString: function() { return mockPrService; },
+      persistEncryptedString: function() { return mockPrService; },
+      updateConfig: function() { return mockPrService; },
+      execute: function(callbacks) {
+        if (callbacks) { mockPrCallback(callbacks); }
+      }
+    };
+    mockPrCallback = function() {};
+
+    $provide.constant('prService', mockPrService);
```
Comment (michaelwehrley) [inline test/unit/controllers/MainControllerSpec.js]: I see :+1:

**Comment by michaelwehrley** [comment]
looks good. :ear:

**Comment by ericcf** [comment]
Finally passed on Travis after problems on their end. Merging.

**Pull request merged**